apps/storybook/src/Slider.stories.tsx

```diff
@@ -65,6 +81,14 @@ MultiThumbsAllowCrossing.args = {
 };
 
 export const WithCustomThumb: Story<SliderProps> = (args) => {
+  const thumbProps = args.thumbProps();
```

Comment:
Here `thumbProps` expects index. So maybe just undo your changes and disable orientation control for this story.
![image](https://user-images.githubusercontent.com/36186912/182164357-cac2fb2d-82a8-497c-8eda-ebc8aef25157.png)

Reply:
I'm not sure why, but I'm not getting that error.

![image](https://user-images.githubusercontent.com/45748283/182509881-aff7e8ae-303a-48aa-917b-67d3ad42df65.png)

Also, `args.thumbProps()` for `WithCustomThumb` does not take any parameter

![image](https://user-images.githubusercontent.com/45748283/182509959-86cc01d5-ce25-4735-b4be-4538dfeffc55.png)

But actually, it might be easier to just allow horizontal orientation for the with custom thumb example. This is because in the horizontal and vertical examples, we need slightly different styling of the thumb. Below is the code example from lines 83-93 from `Slider.stories.tsx`.

```tsx
export const WithCustomThumb: Story<SliderProps> = (args) => {
  const thumbProps = args.thumbProps();
  thumbProps.style = {
    ...thumbProps.style,
    ...(args.orientation === 'horizontal'
      ? { transform: 'translateX(-19.2px)', top: 0 }
      : { transform: 'translate(-25%, 50%)' }),
  };
  args.thumbProps = () => thumbProps;
  return <Slider {...args} />;
};
```

Reply:
I would go with horizontal only in order not to make our stories' code too complicated.

Reply:
**[DONE]** Thanks, I removed the vertical option from `WithCustomThumb`

![image](https://user-images.githubusercontent.com/45748283/182641318-512f0eb0-bcb6-4853-ac6e-c383b0f01fd4.png)